src/communication/network_interface_sr.c

```diff
@@ -10223,3 +10226,190 @@ ssession_stop_attached_threads (void *session)
 {
   session_stop_attached_threads (session);
 }
+
+void
+scdc_get_lsa (THREAD_ENTRY * thread_p, unsigned int rid, char *request, int reqlen)
+{
+  OR_ALIGNED_BUF (OR_INT_SIZE + OR_LOG_LSA_ALIGNED_SIZE) a_reply;
+  char *reply = OR_ALIGNED_BUF_START (a_reply);
+  char *ptr;
+  LOG_LSA start_lsa;
+  time_t input_time;
+  uint64_t b_start_lsa;
+  int error;
```

Comment:
How about 'error_code' or 'return_code' or the name you specified in the protocol specification?